package.json

```diff
@@ -1,9 +1,21 @@
 {
+  "workspaces": [
+    "packages/monthly_roadmap",
+    "packages/github_utils",
+    "packages/markdown",
+    "packages/date_utils"
+  ],
   "devDependencies": {
+    "@biomejs/biome": "1.6.4",
+    "@octokit/rest": "^20.1.0",
+    "@types/github-script": "github:actions/github-script",
     "vitest": "^1.4.0"
   },
   "scripts": {
-    "test": "vitest"
+    "test": "vitest",
+    "lint": "biome lint packages --write",
```

Comment:
The `lint` command does not accept a `--write`, and in all cases I'd suggest to put the flag before the folders - although this is mainly to align with the docs and it'll work either way:

```json
"lint": "biome lint packages",
"format": "biome format --write packages",
```

After the change, running the `format` command causes the imports to be sorted so you might want to run and make another commit it before merging.

Reply:
fixed it :) tks!!